docs/tutorials/csharp/bert-nlp-csharp-console-app.md

```diff
@@ -0,0 +1,376 @@
+---
+title: Inference with C# BERT NLP and ONNX Runtime
```

Comment:
This is where I would add "deep learning" to the meta title and meta description

Reply:
Added. I was concerned about a longer title in the nav. I will see how it looks in the staged site once I check this change in.

Reply:
That's fair